cluster/addons/metadata-agent/stackdriver/metadata-agent-rbac.yaml

```diff
@@ -7,9 +7,7 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
 rules:
 - apiGroups:
-  - ""
-  - "apps"
-  - "extensions"
+  - "*"
```

Comment:
Why do we need such extensive permission for metadata-agent? Do we actually need more than pods and nodes?

Reply:
The metadata agent needs to follow the owner references to find the top-level owner for a given pod. With custom object types and operators, the owner chain may contain anything. Restricting this access will result in incomplete data.